projects/rocprofiler-compute/CHANGELOG.md

```diff
@@ -5,8 +5,10 @@ Full documentation for ROCm Compute Profiler is available at [https://rocm.docs.
 ## Unreleased
 
 ### Added
+* Add `--list-blocks <arch>` option to general options to list available IP blocks on specified arch (similar to `--list-metrics`), cannot be used with `--block`.
 
 ### Changed
+* `-b/--block` accepts block alias(es) (See block aliases using command-line option `--list-blocks <arch>`).
```

Comment:
Even though metrics generation system is transparent to the end user, I'd still suggest mentioning changes here:
Metrics yaml configuration files  are all reformatted and generated with new metrics generation sub-system.